bin/node/e2e-tests/Cargo.toml

```
@@ -0,0 +1,35 @@
+[package]
+name = "node-e2e-tests"
+version = "0.1.0"
+authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2018"
```

let's make this non publish
```suggestion
edition = "2018"
publish =  false
```